R/html_dependencies.R

```diff
@@ -328,14 +343,18 @@ validate_html_dependency <- function(list) {
   if (is.null(list$version))
     stop("version for html_dependency not provided", call. = FALSE)
   list <- fix_html_dependency(list)
-  if (is.null(list$src$file))
-    stop("path for html_dependency not provided", call. = FALSE)
-  file <- list$src$file
-  if (!is.null(list$package))
-    file <- system.file(file, package = list$package)
-  if (!file.exists(file)) {
-    utils::str(list)
-    stop("path for html_dependency not found: ", file, call. = FALSE)
+
+  # check src path or href are given
+  if (is.null(list$src)) {
+    stop("neither path nor href are provided for html_dependency", call. = FALSE)
```

Comment:
Currently we have a lot of `call. = FALSE` in this package. Perhaps we should copy these two helper functions: https://github.com/yihui/knitr/blob/89a1f6cc19e060c0f45b80fe1c4905dfab441193/R/utils.R#L870-L871

Reply:
Yes I was looking for them in **xfun** at first and saw that they are internal to **knitr**. 
I can surely copy them and replace every call !

Reply:
They are one-liners and probably not worth exporting in **xfun**.